src/components/UniversalTable/UniversalTable.js

```diff
@@ -0,0 +1,55 @@
+import "./UniversalTable.css";
+
+export default function UniversalTable(collection, options) {
+  this.rowClick = options.onClick;
+  this.collection = collection;
+  this.parent = options.parent;
+  this.headerCells = options.header;
+}
+
+UniversalTable.prototype.render = function () {
```

Comment:
let's follow the general flow we used everywhere else - leave parent here, in the way that each render could happen anywhere, and the only thin component needs to know it where to put all it's stuff. remove parent property from the constructor, or leave it null until render wasn't called